common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -26,129 +26,145 @@ public class CommonValue {
     public static String WARM_BOX_TYPE = "warm";
 
     public static final String ML_MODEL_INDEX = ".plugins-ml-model";
+    public static final String ML_MODEL_RELOAD_INDEX = ".plugins-ml-model-reload";
+    public static final Integer ML_MODEL_RELOAD_MAX_RETRY_TIMES = 2;
     public static final String ML_TASK_INDEX = ".plugins-ml-task";
+    public static final Integer ML_MODEL_RELOAD_INDEX_SCHEMA_VERSION = 3;
     public static final Integer ML_MODEL_INDEX_SCHEMA_VERSION = 2;
     public static final Integer ML_TASK_INDEX_SCHEMA_VERSION = 1;
     public static final String USER_FIELD_MAPPING = "      \""
-            + CommonValue.USER
-            + "\": {\n"
-            + "        \"type\": \"nested\",\n"
-            + "        \"properties\": {\n"
-            + "          \"name\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\", \"ignore_above\":256}}},\n"
-            + "          \"backend_roles\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}},\n"
-            + "          \"roles\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}},\n"
-            + "          \"custom_attribute_names\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}}\n"
-            + "        }\n"
-            + "      }\n";
+        + CommonValue.USER
+        + "\": {\n"
+        + "        \"type\": \"nested\",\n"
+        + "        \"properties\": {\n"
+        + "          \"name\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\", \"ignore_above\":256}}},\n"
+        + "          \"backend_roles\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}},\n"
+        + "          \"roles\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}},\n"
+        + "          \"custom_attribute_names\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}}\n"
+        + "        }\n"
+        + "      }\n";
+    public static final String NODE_ID_FIELD = "node_id";
+    public static final String MODEL_LOAD_RETRY_TIMES_FIELD = "retry_times";
+
+    public static final String ML_MODEL_RELOAD_INDEX_MAPPING = "{\n"
+        + "    \"properties\": {\n"
+        + "      \""
+        + NODE_ID_FIELD
+        + "\" : {\"type\": \"keyword\"},\n"
+        + "      \""
+        + MODEL_LOAD_RETRY_TIMES_FIELD
+        + "\" : {\"type\": \"long\"}\n"
```

Comment:
we do we use long type?

Reply:
How do we differentiate different model's reload times?

Reply:
it should be "integer" type, I will modify it.

Reply:
> How do we differentiate different model's reload times?

we don't differentiate different model. If a node halt down with unknown reason, it will reload all the model under this stopped node. so what we should do is to reload all the model under broaken node